documentation/modules/exploit/multi/misc/weblogic_deserialize_asyncresponseservice.md

```diff
@@ -39,13 +39,9 @@ msf5 exploit(multi/misc/weblogic_deserialize_asyncresponseservice) > check
   3. Configure the payload and payload parameters.
   4. `run`
 
-## Options
+## Option
```

Comment:
Since this is a header, I would keep this consistently plural, even if there is only one option.